src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -10,22 +10,70 @@
 
 namespace Microsoft.PowerShell.Commands
 {
+    #region Restart-Computer
+
+    /// <summary>
+    /// Cmdlet to restart computer.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Restart, "Computer", SupportsShouldProcess = true,
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2097060", RemotingCapability = RemotingCapability.SupportedByCommand)]
+    public sealed class RestartComputerCommand : CommandLineCmdletBase
+    {
+        // TODO: Support remote computers?
+
+        #region "Overrides"
+
+        /// <summary>
+        /// BeginProcessing.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            RunCommand("/sbin/shutdown", "-r now");
+        }
+        #endregion "Overrides"
+    }
+    #endregion Restart-Computer
+
     #region Stop-Computer
 
     /// <summary>
     /// Cmdlet to stop computer.
     /// </summary>
     [Cmdlet(VerbsLifecycle.Stop, "Computer", SupportsShouldProcess = true,
         HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2097151", RemotingCapability = RemotingCapability.SupportedByCommand)]
-    public sealed class StopComputerCommand : PSCmdlet, IDisposable
+    public sealed class StopComputerCommand : CommandLineCmdletBase
     {
-        #region Private Members
+        // TODO: Support remote computers?
 
-        private Process _process = null;
+        #region "Overrides"
 
-        #endregion
+        /// <summary>
+        /// BeginProcessing.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            var args = "";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                args = "-P now";
+            }
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                args = "now";
+            }
+            RunCommand("/sbin/shutdown", args);
```

Comment:
The style rules we have expect a newline after closing brace
```suggestion

            RunCommand("/sbin/shutdown", args);
```

Reply:
@brendandburns Please address the comment.

Reply:
Done.